### PR TITLE
EULA as a separate file and in the About section

### PR DESCRIPTION
**Pull request by janper**
I've found a template for an EULA. Please, have a look and give your feedback.

The EULA is now in two places - as a part of the **About** window inside the software and as a **separate file** ~`license.md`~ `eula.md`. I'm not sure if the `.md` format is usual an suitable. We should investigate a bit to find the most suitable option for distributing the license agreement.

My findings from free software so far:

- **Blender**: a `copyright.txt` file in the Blender root folder. It seems to be a derivative of a GNU GPL license. There are also many other `.txt` files with licenses for partial things, such as libraries and fonts
- **FileZilla**: a `GPL.html` file in the program's root folder
- **Firefox**: no apparent license file in program's root and subfolders
- **Adobe Reader DC**: a `ReadMe.htm` file in program's root - it's a combination of readme and a brief license (single page, no scrolling needed)
- **VLC**: a `COPYING.txt` file in program's root folder with GNU GPL inside. Many other files, such as `AUTHORS.txt` or `THANKS.txt`

Aside from polishing the license itself (or simply choosing GNU GPL just like anybody else), I would like us (and by "us" I mean @ondrowan ) to implement this:

- [x] Display the license agreement during the installation process
- [x] Save the license file next to the executable binary during the installation

### CODE REVIEW

**Comment by yanchith** [comment]
> The EULA is now in two places - as a part of the About window inside the software and as a separate file license.md

You mean `eula.md`, right?

About your license investigation. Yes - every software does it a bit differently. What we already have (a top level `LICENSE` file) should be sufficient, but if we really want to be bulletproof, the license or a link to it should be included in every source file. This is usually done by build scripts. I suggest we don't do this yet, or hopefully ever, and consider our current solution sufficient.

As for the installer tasks you listed - I'd make that a separate issue in our trello and let Ondro or me create a separate PR for it.

Gonna review the source changes now.

File: src/ui.rs [hunk 1]
```diff
@@ -576,7 +576,103 @@ impl<'a> UiFrame<'a> {
...
+                    \n\
+                    You are permitted to load the H.U.R.B.A.N. selector software (for example a PC, \
+                    laptop, mobile or tablet) under your control. You are responsible for ensuring \
+                    your device meets the minimum requirements of the H.U.R.B.A.N. selector software.\n\
+                    \n\
+                    You are not permitted to:\n\
+                    * Edit, alter, modify, adapt, translate or otherwise change the whole or any part of \
+                    the Software nor permit the whole or any part of the Software to be combined with or \
+                    become incorporated in any other software, nor decompile, disassemble or reverse \
+                    engineer the Software or attempt to do any such things\n\
```
Comment (yanchith) [inline src/ui.rs]: Not sure this clause makes sense. To me it looks to be in direct violation with the GNU GPL. People can modify and adapt the software. We already provide the source they can build from, so no need to decompile anything. Is this clause necessary?

If we modify this clause, let's not forget to modify the copy of it in `eula.md`

File: eula.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# End-User License Agreement (EULA) of H.U.R.B.A.N. selector
```
Comment (yanchith) [inline eula.md]: Could you please reflow the text in this file so that the longest line is shorter than 100 chars?

**Review by yanchith — COMMENTED** [review]
Nice! See inline

File: src/ui.rs (hunk 1 above)
Comment (janper) [reply src/ui.rs]: okay, i'll remove it

**Review by yanchith — DISMISSED** [review]

**Comment by janper** [comment]
Too soon for the approval. Mainly we need to make sure the text is what we want it to be. 
Lucia says:

```
Chapem to teda vseobecne takto:

- Intelektualne vlastnictvo na softver - subdigital
- Prava na pouzivanie softvera - SCD

Avsak, povodne sme zamyslali softver vydat ako Open source. Nejde potom o upravovanie toho isteho vztahu? Potrebujeme aj Open source aj EULA licenciu? Resp. ich kombinaciu?

Nasla som k tomu na Githube EULA, ktora ma v sebe zahrnutu aj sucinnost s Open source:
https://desktop.github.com/eula/

Zuzana Motuzova (https://www.akmotuzova.sk/) poskytuje v tejto oblasti poradenstvo, neviem ci aj bezplatne ale spolupracuje vela s Progressbarom napriklad. Mozem ju kontaktovat...
```

**Comment by yanchith** [comment]
> Too soon for the approval.

Yep, I've seen that. I merely mean that I don't have any further objections here.

**Comment by janper** [comment]
yup. but i'll need some help here. can't decide on my own.

File: eula.txt [hunk 3]
```diff
@@ -26,18 +27,18 @@ or if you do not agree with the terms and conditions of this EULA agreement, do
 not install or use the Software, and you must not accept this EULA agreement.
 
 This EULA agreement shall apply only to the Software supplied by Slovak Design
-Center and Subdigital s.r.o. herewith regardless of whether other software is
-referred to or described herein. The terms also apply to any Slovak Design
-Center and Subdigital s.r.o. updates, supplements, Internet-based services, and
-support services for the Software, unless other terms accompany those items on
-delivery. If so, those terms apply. This EULA was created by EULA Template for
-H.U.R.B.A.N. selector.
+Center. herewith regardless of whether other software is referred to or
```
Comment (yanchith) [inline eula.txt]: Forgot to remove a dot here, after "Center."

File: eula.txt [hunk 4]
```diff
@@ -26,18 +27,18 @@ or if you do not agree with the terms and conditions of this EULA agreement, do
...
+described herein. The terms also apply to any Slovak Design Center. updates,
```
Comment (yanchith) [inline eula.txt]: And here, after "Center."

File: eula.txt [hunk 5]
```diff
@@ -26,18 +27,18 @@ or if you do not agree with the terms and conditions of this EULA agreement, do
...
+This EULA was created by EULA Template for H.U.R.B.A.N. selector.
 
-## License Grant
+License Grant
+-------------
 
-Slovak Design Center and Subdigital s.r.o. hereby grants you a personal,
-non-transferable, non-exclusive licence to use the H.U.R.B.A.N. selector
-software on your devices in accordance with the terms of this EULA agreement.
+Slovak Design Center. hereby grants you a personal, non-transferable,
```
Comment (yanchith) [inline eula.txt]: dot

File: eula.txt [hunk 6]
```diff
@@ -64,22 +66,24 @@ property rights of whatever nature in the Software, including any modifications
 made thereto) are and shall remain the property of Slovak Design Center and
 Subdigital s.r.o..
 
-Slovak Design Center and Subdigital s.r.o. reserves the right to grant licences
-to use the Software to third parties.
+Slovak Design Center. reserves the right to grant licences to use the Software
```
Comment (yanchith) [inline eula.txt]: dot

**Review by yanchith — COMMENTED** [review]
Nice, just remove the forgotten dots.

**Review by yanchith — DISMISSED** [review]

**Review by yanchith — APPROVED** [review]

**Pull request merged**